### PR TITLE
creacion de sitom v2

### PR DESCRIPTION
**Pull request by omolds**
creando el sources de sitom v2 para omolds

### CODE REVIEW

**Comment by omolds** [comment]
I signed it!

**Comment by omolds** [comment]
I signed it!

**Comment by omolds** [comment]
I signed it!

File: sources/sitomv2.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "id": "SITOMV2",
+  "name": "SITOMV2",
+  "categories": ["ANALYTICS", "ADVERTISING", "DATABASE"],
+  "organization": "Omolds",
```
Comment (pfrisella) [inline sources/sitomv2.json]:
```suggestion
  "organization": "OMOLDS",
```

File: sources/sitomv2.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "id": "SITOMV2",
+  "name": "SITOMV2",
```
Comment (pfrisella) [inline sources/sitomv2.json]: This is the product/source name that will be shown to users, is this correct or should it just be SITOM ?

Comment (omolds) [reply sources/sitomv2.json]: SITOM is the name of the product

**Comment by omolds** [comment]
I signed it!

**Comment by pfrisella** [comment]
CLA issue needs to be resolved.
Closing due to inactivity.

**Pull request closed without merging**